linux-tkg-config/prepare

```diff
@@ -554,29 +515,26 @@ _tkg_srcprep() {
     cat "${_where}/${_configfile}" > ./.config
   fi
 
+  if [ "${_distro}" = "Arch" ]; then
+    # Reset local version string if ever it's in the .config file
+    scripts/config --set-str localversion ""
```

Comment:
I still fail to see the point of this when we are running `make LOCALVERSION=` in the PKGBUILD. Were your tests on Arch done with the PKGBUILD or with install.sh regarding that versioning duplication problem?

Reply:
I did it on Arch, I think it is reproducible if you have a config file where `LOCALVERSION` is explicitly set or you set it with `make xconfig`.